EIPS/eip-1.md

```diff
@@ -79,15 +79,15 @@ The following is the standardization process for all EIPs in all tracks:
 
 **Review** - An EIP Author marks an EIP as ready for and requesting Peer Review.
 
-**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later.
+**Last Call** - This is the final review window for an EIP before moving to `Final`. An EIP editor will assign `Last Call` status and set a review end date (`last-call-deadline`), typically 14 days later. If 6 months pass after the `last-call-deadline`, it is moved back into `Review`.
```

Comment:
Should we move it to `Stagnant` directly?

Reply:
Last call should imply some degree of community participation. If you want, this time can be reduced down to e.g. one month. But -1 to moving directly to Stagnant.